gatorgrade/output/output_tools.py

```diff
@@ -1,3 +1,7 @@
 # This file is used for storing supporting functions and tools that are used by the methods in output_functions.

```

Comment:
Please add a docstring to the top of this file, describing what the contents of this file will accomplish.